test/parallel/test-http2-client-socket-operations.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+const h2 = require('http2');
+
+const server = h2.createServer();
+server.on('stream', common.mustCall((stream) => {
+    stream.respond();
+    stream.end();
+  })
+);
+
+server.listen(0, common.mustCall(() => {
+    const client = h2.connect(`http://localhost:${server.address().port}`);
+    const req = client.request({ ':path': '/' });
+
+    req.on(
+      'response',
+      common.mustCall(() => {
```

Comment:
Same as above, indent level is off now. I don't think this will pass the linter.

Nit: this would normally be one line in node `req.on('response', common.mustCall(() => {`. Same below for `end`.

Reply:
Adjusted indentation and confirmed that `make lint-js` passes in new commit